### ISSUE TITLE
Launch minimized not working

### ISSUE THREAD

**Issue by WebShapedBiz** [timche/meru#297]
**OS**
Linux, elementaryOS 6 Odin

**Gmail Desktop Version**
3.0.0-alpha.27 (dev)

**Describe the bug**
Although I have "Launch Minimized" selected, Gmail Desktop is always autostarting maximized.
The `.desktop` file in `.config/autostart` folder doesn't have any parameters weather I select "Launch minimized" or not:
It's always `Exec="/opt/Gmail Desktop/gmail-desktop" %U`

Is there any parameter that I could add manually?

Thank you for the time and effort you've put into this gem.

Cheers.

**Comment by DinCahill** [comment]
I tried elementaryOS 6.1 in a virtual machine, and it worked for me with the deb package.

Here's what I did:

- Downloaded the [3.0.0-alpha.27 deb](https://github.com/timche/gmail-desktop/releases/download/v3.0.0-alpha.27/gmail-desktop-3.0.0-alpha.27-linux.deb) and installed it with `sudo dpkg -i gmail-desktop-3.0.0-alpha.27-linux.deb`.
- (Optional) Manually compiled and installed [wingpanel-indicator-na-tray](https://github.com/msmaldi/wingpanel-indicator-na-tray) to get the tray icon. I had to change `wingpanel-2.0` to `wingpanel` in `meson.build` to get it to build on Odin. 
- Added Gmail Desktop to startup using elementaryOS Settings → Applications → Startup.
- Ran Gmail Desktop and enabled the Launch Minimized setting.

https://user-images.githubusercontent.com/510163/149655581-ee03508d-69c6-429d-a6f3-a02337c38b8b.mp4

**Comment by WebShapedBiz** [comment]
Thank you very much @DinCahill for your time and effort here, you really made an effort. I've done everything from your list, except compiling myself, but with no luck. I'll try to find out why I have this issue on both my computers while everything works as it should on yours.

 Thank you again for the help.

Cheers.

**Comment by timche** [comment]
@WebShapedBiz is this still an issue with the latest version [v3.0.0-alpha.31](https://github.com/timche/gmail-desktop/releases/tag/v3.0.0-alpha.31)? Unfortunately I can't reproduce it on macOS, but Gmail Desktop accepts `launch-minimized` as command line argument.

**Comment by WebShapedBiz** [comment]
Even worse @timche, now it doesn't even start. This is the error from the terminal:
```
15:42:27.229 › Checking for update
mesa: for the --simplifycfg-sink-common option: may only occur zero or one times!
mesa: for the --global-isel-abort option: may only occur zero or one times!
mesa: for the --amdgpu-atomic-optimizations option: may only occur zero or one times!
[65909:0212/154227.246502:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
15:42:27.248 › Error: Error: ENOENT: no such file or directory, open '/opt/Gmail Desktop/resources/app-update.yml'
(node:65875) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/opt/Gmail Desktop/resources/app-update.yml'
(Use `gmail-desktop --trace-warnings ...` to show where the warning was created)
(node:65875) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

```

**Comment by timche** [comment]
@WebShapedBiz can you please try to re-install the app please? Unfortunately I'm not using Linux myself, so it's very difficult for me to see what exactly is wrong or caused it.

**Comment by WebShapedBiz** [comment]
I've installed version alpha.29, alpha.30, and alpha.31, and on each version that app starts but with empty Gmail account tabs. After that, I've reinstalled version alpha.27 that I've been using all this time and everything is back to normal.

**Comment by WebShapedBiz** [comment]
New development - on my first email account everything is fine but on the second account, I cannot log in due to the "This browser or app may not be secure" error. It's hilarious because on both accounts I'm not allowing unsecured apps.

**Comment by timche** [comment]
@WebShapedBiz you can try with disabling "Use User Agent Fix" or try signing in multiple times. This "security blocker" is happening very inconsistently.

**Comment by WebShapedBiz** [comment]
Thank you it worked :-)

**Comment by timche** [comment]
@WebShapedBiz so do you still have the original issue though?

**Comment by WebShapedBiz** [comment]
Yes I do and unfortunately haven't had a chance to test new alpha versions
because they won't run.

Best regards,


*Siniša Perović**Webshaped Business OÜ*


***@***.*** ***@***.***>*


*https://www.webshaped.biz <https://www.webshaped.biz/>
***@***.***>+385 92 1244627*

**Comment by DinCahill** [comment]
@WebShapedBiz Have you tried running the latest version with a clean config?

`mv ~/.config/Gmail\ Desktop ~/.config/Gmail\ Desktop.bak`

**Comment by WebShapedBiz** [comment]
Well, this will amuse you.

On my main PC, where I tried yesterday new alpha releases and they didn't work, I reinstall back alpha.27 version, which works fine but also, after all this installation and reinstallation gmail-desktop now actually starts minimized. I haven't touch preferences at all.
On my laptop, same OS, same apps, alpha.32 with clean config works fine BUT I cannot get it to start minimized. Go figure.

Anyhow, thank you very much @DinCahill for your time and effort.

Cheers.

**Comment by serhumanos** [comment]
I have the same Issue

KDE Neon (Ubuntu 20.04)
deb package
